tmt/utils.py

```diff
@@ -986,7 +999,11 @@ def store_cli_invocation(cls, context: 'tmt.cli.Context') -> 'tmt.cli.CliInvocat
             raise GeneralError(
                 f"{cls.__name__} attempted to save a second CLI context: {cls.cli_invocation}")
 
-        cls.cli_invocation = tmt.cli.CliInvocation.from_context(context)
+        if options:
```

Comment:
`if options is None` would be better: the default value being `None`, empty (and therefore false-ish) `options` dict looks like an acceptable set of options.

Reply:
Ack, updated in 93c1eec1.